examples/cryptorealtime/terraform-setup/main.tf

```diff
@@ -0,0 +1,152 @@
+provider "google" {
```

Comment:
Copyright header at the top of the file? (And other *.tf files as well)

Reply:
.tf is not recognized with autogen 
File type not recognized: terraform-setup/main.tf

Reply:
just FYI

Reply:
You can easily generate copyright headers using a different filetype that has the same comment format, e.g., `autogen foo.sh` (since Terraform accepts `#` as a comment character). In any case, I've added Terraform support recently so you can use it now with the same filename, and you can use the `-i` flag to add the comments to an existing file in-place.

Please add comment headers to all other files as well, where applicable.